GUI/QtQuick/qml/StyledControls/CopyIconButton.qml

```diff
@@ -0,0 +1,43 @@
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import "../BsStyles"
+
+Image {
+    id: control
+
+    signal copy();
+
+    x: internal_text.contentWidth + copy_icon.width / 2
+    width: 24
+    height: 24
+    anchors.verticalCenter: parent.verticalCenter
+    source: "qrc:/images/copy_icon.svg"
+    MouseArea {
+        anchors.fill: parent
+        ToolTip {
+            id: tool_tip
+            timeout: 1000
+            text: qsTr("Copied")
+            font.pixelSize: 10
+            font.family: "Roboto"
+            font.weight: Font.Normal
+            contentItem: Text {
+                text: tool_tip.text
+                font: tool_tip.font
+                color: BSStyle.textColor
+            }
+            background: Rectangle {
+                color: BSStyle.buttonsStandardColor
+                border.color: BSStyle.buttonsStandardColor
+                border.width: 1
+                radius: 14
+            }
+        }
+        onClicked: {
+            control.copy()
+            tool_tip.visible = true
+        }
+    }
+}
```

Comment:
One more thing to fix in idle mode is a final linefeed (at end of file). Here's how github complains about the miss of it:
![image](https://user-images.githubusercontent.com/17161640/216917074-f700cd33-fc88-4ea0-8d70-8bc401d1cb58.png)